apps/foundation-demo/src/app/design/molecules/sidebar/sidebar-viewport/sidebar-viewport.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, Output, EventEmitter, Input, ChangeDetectionStrategy } from '@angular/core';
+
+import { DaffSidebarMode } from '../helper/sidebar-mode';
+import { daffSidebarAnimations } from './sidebar-viewport.animation';
+
+@Component({
+  selector: 'daff-sidebar-viewport',
+  templateUrl: './sidebar-viewport.component.html',
+  styleUrls: ['./sidebar-viewport.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  animations: [
+    daffSidebarAnimations.transformSidebar,
+    daffSidebarAnimations.transformContent,
+    daffSidebarAnimations.fadeBackdrop
+  ]
+})
+export class DaffSidebarViewportComponent{
+
+
+  _animationState: string = "void";
+
+  /**
+   * Property for the "opened" state of the sidebar
+   */
+  @Input()
+  get opened(): boolean { return this._opened; }
+  set opened(value: boolean) { 
+    if(this._opened == value){return;}
+    this._opened = value;
+    this._openedChanged.emit(value);
+    this._animationState = getAnimationState(value);
```

Comment:
Is it possible to change whether you want animations or not? Looks like you can't toggle it off

Reply:
@lderrickable Let me know if the `side` mode meets this goal.

Reply:
I would have thought that the purpose of having an `animationsEnabled` variable is to be able to have no TIMED animations on opening and closing the sidebar. If we're not giving the user that choice, I think we should just remove the whole `animationsEnabled` thing altogether. Even if animations are enabled all the time, they shouldn't have an effect on the `SIDE` mode version.